pkgs/top-level/python-aliases.nix

```diff
@@ -0,0 +1,37 @@
+lib: pythonPackages:
+
+with pythonPackages;
+
+let
+  # Removing recurseForDerivation prevents derivations of aliased attribute
+  # set to appear while listing all the packages available.
+  removeRecurseForDerivations = alias: with lib;
+    if alias.recurseForDerivations or false then
+      removeAttrs alias ["recurseForDerivations"]
+    else alias;
```

Comment:
```suggestion
  removeRecurseForDerivations = alias: removeAttrs alias [ "recurseForDerivations" ];
```

Reply:
I am not sure about the internals of pythonPackages. Fridh self assigned this so I leave it up to him.